timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md

```diff
@@ -137,7 +137,31 @@ unblocks other transactions, and materializes the aggregates. The first
 transaction is very quick, and most of the work happens during the second
 transaction, to ensure that the work does not interfere with other operations.
 
+## Using continuous aggregates in a multi-node environment
+If you are using Timescale in a multi-node environment, there are some
+additional considerations for continuous aggregates.
+
+When you create a continuous aggregate within a multi-node environment, you must
+create the continuous aggregate on the access node. While it is possible to
+create a continuous aggregate on data nodes, it does not work as expected, and
+could cause problems within your multi-node environment.
+
+When you refresh a continuous aggregate on an access node, it computes a single
+window to update the time buckets. This could slow down your query, especially
+if your data nodes are remote.
```

Comment:
Added some clarifications. The connection between a single window and why this would make it slow is not clear.

```suggestion
When you refresh a continuous aggregate on an access node, it computes a single
window to update the time buckets. This could slow down your query if the actual number of rows that were updated is small, but widely spread apart. This is aggravated if the network latency is high if, for example, you have remote data nodes.
```